config_d/registers

```diff
@@ -40,6 +40,10 @@ BLOCKS_COMPAT_VERSION = 0
     PCAP_START_WRITE        8
     PCAP_WRITE              9
 
+    # Hardware timestamps
+    PCAP_TS_SEC             10
+    PCAP_TS_NSEC            11
```

Comment:
Just realised that this will be in ticks not in nanoseconds, so we'll have to do the scaling on the server
```suggestion
    PCAP_TS_TICKS            11
```